subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/declaring_repositories.adoc

```diff
@@ -490,22 +490,52 @@ As the way to declare the repositories and what they are expected to contain dep
 [[sub:centralized-repository-declaration]]
 == Centralizing repositories declaration
 
-Instead of declaring repositories in every subproject of your build or via an `allprojects` block, Gradle offers a way to declare them in a central place for all project.
+Instead of declaring repositories in every subproject of your build or via an `allprojects` block, Gradle offers a way to declare them in a central place for all projects.
 
-NOTE: Central declaration of repositories is an incubating feature
+NOTE: Central declaration of repositories is an incubating feature.
 
-Repositories used by convention by every subproject can be declared in the `settings.gradle(.kts)` file:
+Repositories used by convention in every subproject can be declared in the `settings.gradle(.kts)` file:
 
 .Declaring a Maven repository in settings
 ====
 include::sample[dir="snippets/artifacts/defineRepositoryInSettings/kotlin",files="settings.gradle.kts[tags=declare_repositories_settings]"]
 include::sample[dir="snippets/artifacts/defineRepositoryInSettings/groovy",files="settings.gradle[tags=declare_repositories_settings]"]
 ====
 
-The `dependencyResolutionManagement` repositories block accepts the same notations as in a project, which includes Maven or Ivy repositories, with or without credentials, etc.
+The `dependencyResolutionManagement` repositories block accepts the same notations as in a project. This includes Maven or Ivy repositories, with or without credentials, etc.
 
 By default, repositories declared by a project will *override* whatever is declared in settings.
-You can change this behavior to make sure that you always use the settings repositories:
+This means that repositories declared in a project `build.gradle(.kts)` file *override* those declared in the `settings.gradle(.kts)` file:
```

Comment:
I think the second sentence is a little redundant and doesn't add much over the previous sentence aside from that it also mentions the build/settings file names. You could drop it or merge it.